docs/03-tutorials/00-observability/obsv-03-define-alerting-rules-monitor.md

```diff
@@ -71,4 +71,7 @@ You have performed the steps to observe application metrics using the `monitorin
 
    ![Alert on the dashboard](./assets/fired-alert.png)
 
-6. If you don't want to proceed with the following tutorial, [clean up the configuration](obsv-05-clean-up-configuration.md).
+## Next steps
+
+- Follow the tutorial to [Send notifications](obsv-04-send-notifications.md).
+- If you don't want to proceed with the following tutorial, [clean up the configuration](obsv-05-clean-up-configuration.md).
```

Comment:
```suggestion
- If you don't want to proceed with the tutorial, [clean up the configuration](obsv-05-clean-up-configuration.md).
```

Reply:
Mhm, we're really talking about the subsequent set of instructions here.
As I understood it, in Kyma the single sets of instructions are called "tutorials", and a set of them is a "tutorial flow"?

Reply:
But you're referring to the tutorial mentioned in the line above, right? That's why `following` didn't sound right for me here 🤔